Remove informationSegment as parameter (#257)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md

Get-SPOSite -Identity https://contoso.sharepoint.com/sites/research | Select InformationSegment
```

> This example returns the InformationSegments associated to the site. It is applicable for tenants who have enabled Microsoft 365 Information barriers capability. Please read https://docs.microsoft.com/microsoft-365/compliance/information-barriers documentation to understand Information barriers in SharePoint Online.

Reviewer: ```suggestion
This example returns the InformationSegments associated with the site. It is applicable for tenants who have enabled Microsoft 365 Information barriers capability. Read [Learn about information barriers](/microsoft-365/compliance/information-barriers) to understand Information barriers in SharePoint Online.
```
